frontend/storage/common.ts

```diff
@@ -7,6 +9,8 @@ export interface AppStore {
 	headerBreadcrumb: string;
 	gamesVisibility: Map<number, boolean>;
 	games: Map<number, Game>;
+	gameSeries: Map<number, GameSeries>;
+	images: Map<number, File>;
```

Comment:
Actually you know what, I'll change `File` to `Image` in the backend since I don't think we'll ever need non-image files and if we do there are ways to go about it.